src/tests/ftest/util/server_utils.py

```diff
@@ -35,9 +35,14 @@
 import fcntl
 import errno
 import yaml
+import getpass
 
-from avocado.utils import genio
+from agent_utils import node_setup_okay, NodeListType
```

Comment:
(pylint-no-name-in-module) No name 'node_setup_okay' in module 'agent_utils'
(pylint-no-name-in-module) No name 'NodeListType' in module 'agent_utils'